renderdoc/driver/gl/gl_driver.cpp

```diff
@@ -1857,14 +1915,22 @@ bool WrappedOpenGL::EndFrameCapture(void *dev, void *wnd)
       }
 
       {
-        RDCDEBUG("Getting Resource Record");
-
-        GLResourceRecord *record = m_ResourceManager->GetResourceRecord(m_ContextResourceID);
-
-        RDCDEBUG("Accumulating context resource list");
-
+        RDCDEBUG("Accumulating context resource list for %d contexts",
+                 m_ActiveContextResourceIDs.size());
```

Comment:
`%d` isn't the specifier for a `size_t` - either `%zu` (custom renderdoc extension) or need to cast to uint32/uint64 and use their specifiers.